v3/internal/runtime/desktop/@wailsio/runtime/src/create.ts

```
@@ -8,37 +8,27 @@
 }
 
 /**
  * Array takes a creation function for an arbitrary type
  * and returns an in-place creation function for an array
  * whose elements are of that type.
- * @template T
- * @param {(source: any) => T} element
- * @returns {(source: any) => T[]}
  */
-export function Array(element) {
+export function Array<T>(element: (source: any) => T): (source: any) => T[] {
```

Biome (1.9.4)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)